central/reports/service/v2/convert.go

```diff
@@ -133,10 +133,13 @@ func (s *serviceImpl) convertV2NotifierConfigToProto(notifier *apiV2.NotifierCon
 			Id: notifier.GetEmailConfig().GetNotifierId(),
 		},
 	}
-	if notifier.GetEmailConfig() != nil {
+
+	if emailConfig := notifier.GetEmailConfig(); emailConfig != nil {
 		ret.NotifierConfig = &storage.NotifierConfiguration_EmailConfig{
 			EmailConfig: &storage.EmailNotifierConfiguration{
-				MailingLists: notifier.GetEmailConfig().GetMailingLists(),
```

Comment:
Add notifier id

Reply:
For storage proto, we do not store notifier id in the `EmailNotifierConfiguration`, but we added a separate `one of` field called `Ref` inside `NotifierConfiguration`. The notifier id is assigned to `Ref` on ln133 above.